R/Application.R

```diff
@@ -325,43 +304,30 @@ Application = R6::R6Class(
 
       return(pid)
     },
-    #------------------------------------------------------------------------
     print_endpoints_summary = function() {
       if (length(self$endpoints) == 0) {
         self$logger$warn("", context = "'Application' doesn't have any endpoints")
       }
       self$logger$info("", context = list(endpoints = self$endpoints))
       return(invisible(self))
     },
-    #------------------------------------------------------------------------
-    add_openapi = function(path = "/openapi.yaml", openapi = openapi_create(),
-                           file_path = "openapi.yaml") {
+    add_openapi = function(path = "/openapi.yaml", file_path = "openapi.yaml") {
       checkmate::assert_string(file_path)
       file_path = path.expand(file_path)
```

Comment:
Must assert that the `file_path` exists and have yml extension.
```r
checkmate::assert_file_exists(file_path, extension = c("yaml", "yml"))
```
Also we must sure that `path` is string.